payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -67,12 +62,7 @@ internal class ElementsSessionJsonParser(
 
         return if (stripeIntent != null) {
             ElementsSession(
-                linkSettings = ElementsSession.LinkSettings(
-                    linkFundingSources = jsonArrayToList(linkFundingSources),
-                    linkPassthroughModeEnabled = linkPassthroughModeEnabled,
-                    linkFlags = linkFlags,
-                    disableLinkSignup = disableLinkSignup,
-                ),
+                linkSettings = parseLinkSettings(linkSettings, linkFundingSources),
```

Comment:
I don't fully understand why we pass in `linkFundingSources` here if it's a subset of `linkSettings`. It feels like the funding sources should be pulled out of the settings object within `parseLinkSettings`, but maybe I'm missing something!

Reply:
We already compute the `linkFundingSources` a little earlier and pass them into `parseStripeIntent`. Doing it like this so we don’t have to redo that operation.

Reply:
Ahh yep, I missed the `parseStripeIntent` part! 👍